Gek62/hometask-5/5.4.9.sql

```diff
@@ -0,0 +1,6 @@
+SELECT actor.first_name, actor.last_name, COUNT(film_actor.film_id)
+FROM actor 
+INNER JOIN film_actor ON actor.actor_id = film_actor.actor_id 
+GROUP BY actor.first_name, actor.last_name 
```

Comment:
What if there are two or more actors with the same first and last name? By the way, it's exactly the case in this database.
So I think it's better to group by actor ID, do you agree?